.github/workflows/publish-release.yml

```diff
@@ -18,6 +18,18 @@ jobs:
         with:
           distribution: 'adopt'
           java-version: 11
+      - name: Build with Gradle
+        env:
+          packageUser: ${{ github.actor }}
+          packagePAT: ${{ secrets.GITHUB_TOKEN }}
+        run: ./gradlew build -x check -x test
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@master
+        with:
+          scan-type: 'fs'
+          scan-ref: '/github/workspace/ballerina/lib'
```

Comment:
If this module does not pack 3rd party dependencies, this also should be as follows.
```suggestion
          scan-ref: '/github/workspace/ballerina'
```
Otherwise the rest of the `scan-ref` fields should be updated as: `scan-ref: '/github/workspace/ballerina/lib'`